MixItUp.Desktop/Services/GawkBoxService.cs

```diff
@@ -84,7 +82,13 @@ public class GawkBoxService : IGawkBoxService
         private async void WebSocket_OnDisconnectOccurred(object sender, System.Net.WebSockets.WebSocketCloseStatus e)
         {
             GlobalEvents.ServiceDisconnect("GawkBox");
-            await this.webSocket.Reconnect();
+
+            do
+            {
```

Comment:
Because these reconfigure "this.Client" and, well WebSocketClientBase doesn't know what the "this.Client" is in this case? If anything we could maybe refactor into MixerWebSocketWrapper? But I was trying a min-fix to see if we can fix the problem before doing any bigger refactors.